Preparing test test_single_fail_no_retry_report_failure
OperationNumber: Op1
Dependencies: 0
MaxRetries: 0
NumberOfFailures: 1
CallDelays: empty
RetryDelay: 0
RevertMaxRetries: 0
RevertNumberOfFailures: 0
RevertCallDelay: empty
RevertRetryDelay: 0
ValidateFunctionResults: empty
RevertValidateFunctionResults: empty
CallCounter: 0
RevertCallCounter: 0
ValidateCallCounter: 0
RevertValidateCallCounter: 0
Deactivate: empty
ThrowException: empty
ValidateThrowException: empty
RevertThrowException: empty
RevertValidateThrowException: empty
HasReportFail: True
HasRevert: False
HasValidate: False
HasRevertValidate: False

Calling Op1: Success False
OnFailedCallbackAsync: Fail.
Run Log:
[*time*][Op1]: Start Executing Op1
[*time*][Op1]: Op1 Failed. Retries exhausted.
[*time*][Op1]: Start Executing Revert Op1
[*time*][Op1]: No undo operation for Op1. Marking as reverted


OnSagaCompleted: Id: test Status: Reverted

*** Telemetry ***
1: StartSaga - SagaID: test, Type: SagaOperations
2: StartOperation - SagaID: test, Operation: Op1
3: EndOperation - SagaID: test, Operation: Op1, Outcome: Failed
4: CustomEvent - SagaID: test, Event: SagaFailure, Properties: None
5: EndSaga - SagaID: test, Outcome: Reverted
